packages/admin/src/stack/Switch.tsx

```diff
@@ -101,9 +105,9 @@ const StackSwitchInner: React.RefForwardingComponent<IStackSwitchApi, IProps & I
     const getTargetUrl = React.useCallback(
         (pageName: string, payload: string, subUrl?: string): string => {
             if (isInitialPage(pageName)) {
-                return match.url;
+                return removeTrailingSlash(match.url);
             } else {
-                return `${match.url}/${payload}/${pageName}${subUrl ? `/${subUrl}` : ""}`;
+                return `${removeTrailingSlash(match.url)}/${payload}/${pageName}${subUrl ? `/${subUrl}` : ""}`;
```

Comment:
```suggestion
                return `${match.url}${payload}/${pageName}${subUrl ? `/${subUrl}` : ""}`;
```

Why don't you simply remove the trailing slash after `match.url`? Is because `match.url` sometimes has a trailing slash, but not always?

Reply:
> Is because `match.url` sometimes has a trailing slash, but not always?

Exactly. For example, if you have a nested Stack. Then the `match.url` on the second level looks like this: `/test/page2` -> has no trailing slash

Also, I think it's more explicit and easier to read if the slash is removed by function call